lib/views/help/how.html.erb

```diff
@@ -6,11 +6,11 @@
     <a href="#title">#</a>
   </h1>
   <p>
-    WhatDoTheyKnow is administered by a small group of dedicated
+    WhatDoTheyKnow is administered by a small team of mySociety staff and dedicated
     <a href="<%= help_credits_path(:anchor => 'volunteers') %>">volunteers</a>
     who have extensive knowledge and experience of Freedom of Information, and
     who support it. Decisions about the administration of the site are taken by
-    these volunteers, with support from mySociety’s Chief Executive and
+    this administration team, with support from mySociety’s Chief Executive and
```

Comment:
LGTM 👍 

I wonder if we could perhaps simplify decision making to something like _"The WhatDoTheyKnow Support Team makes decisions with support from mySociety’s Chief Executive and Trustees."_? Happy to go with the original wording though.

Reply:
I'm happy with @MattK1234's changes as-is 😄

Reply:
I'm similarly happy - it works, let's do it! 👍